packages/mock-addon/src/utils/request.js

```diff
@@ -6,11 +6,14 @@ export function Request(input, options = {}) {
         this.url = input.url;
         this.body = options.body || input.body || null;
         this.signal = options.signal || input.signal || null;
+        this.ignoreQueryParams =
+            options.ignoreQueryParams || input.ignoreQueryParams || false;
```

Comment:
Do we need this value in a request object?

Reply:
I added the `ignoreQueryParams` option to the request object in order to be able to ignore query params on a per-request basis.
In my projects, I would mostly ignore query params globally.
However, it might happen that I need query params to return different values for the same endpoint without going the full route and defining a custom response function.
If you prefer, we can keep the global option only. :smile:

Reply:
Let's keep it global option for now

Reply:
Alright, I updated the PR to keep only the global option.